Create ai-and-ml-overview.md (#119)
MicrosoftDocs/azure-aks-docs: articles/aks/ai-and-ml-overview.md

> ## AI/ML Workloads Overview 

Reviewer: Wondering if somewhere throughout this we want to include anything similar to what Amazon has done [here](https://docs.aws.amazon.com/eks/latest/userguide/machine-learning-on-eks.html)? Couldn't find an equivalent for AI but wanted to share this anyway

Author: I like that suggestion t but I think it would repetitive to some of the content we have in the body of the page. 

Lets me think about if there's a place for both - which may require a reorganization of the content.

> ## AI/ML Workloads Overview 
> Artificial intelligence (AI) and machine learning (ML) are transforming the way we solve complex problems and deliver value to our customers. Generative AI has produced a paradigm shift that has transformed applications into data-driven, personalized experiences across a full range of applications involving text, code, images, videos, voice, music, and more. In doing so, we have also seen an increase in model size, operational complexity, and an increased need for security and cost-efficiency.  

Reviewer: ```suggestion
---
title: AI and ML workloads in Azure Kubernetes Service (AKS)
description: Learn about running AI and ML workloads in Azure Kubernetes Service (AKS).
ms.topic: overview
ms.service: azure-kubernetes-service
ms.date: 12/19/2024
author: schaffererin
ms.author: colinmixon
---

# AI and ML workloads in Azure Kubernetes Service (AKS)

This article provides an overview of running artificial intelligence (AI) and machine learning (ML) workloads in Azure Kubernetes Service (AKS).

## AI and ML today

Artificial intelligence (AI) and machine learning (ML) are transforming the way we solve complex problems and deliver value to customers. Generative AI has produced a paradigm shift that has transformed applications into data-driven, personalized experiences across a full range of applications involving text, code, images, videos, voice, music, and more. We've also seen an increase in model size, operational complexity, and an increased need for security and cost-efficiency.  
```

> To develop cutting-edge AI models, customers need to be able to allocate computing resources across diverse workloads. This includes not only model training but also serving, inference, and managing auxiliary tasks across infrastructure and workflow orchestration. 

Reviewer: ```suggestion
To develop cutting-edge AI models, you need to be able to allocate computing resources across diverse workloads. This includes model training, serving, inference, and managing auxiliary tasks across infrastructure and workflow orchestration. 
```

> ## Challenges of AI / Workloads
> To develop AI / ML applications effectively, it is essential to enable easy integration with open-source software, frameworks, and data platforms. Developing, refining, optimizing, deploying, and monitoring ML models can be challenging and complex. These inherent complexities require a platform, like AKS, that allows you to maximize the performance of core AI workloads while ultimately reducing inefficiencies and bottlenecks historically present when developing and deploying AI and ML workloads. 

Reviewer: ```suggestion
It's essential to enable easy integration with open-source software, frameworks, and data platforms. Developing, refining, optimizing, deploying, and monitoring ML models can be challenging and complex. These inherent complexities require a platform, like AKS, that allows you to maximize the performance of your AI and ML workloads while reducing inefficiencies and bottlenecks.
```

> ## Developing and Deploying AI and ML applicaitons with AKS
> Using AKS to host your AI and ML applications enables you to leverage high-performance infrastructure, cost-efficiencies, and robust security measures while maintaining the pace of innovation. Plus, AKS is an ideal platform for deploying and managing containerized applications that require high availability, scalability, and portability through the use of open-source tools and integration with existing DevOps processes. Azure Kubernetes Service reduces the operational burden of running ML workloads that are optimized for the computational and infrastructure load induced by developing and deploying AI applications. To support this, multiple best practice guides are available to help you deploy robust AI and ML solutions using AKS. 

Reviewer: ```suggestion
## Develop and deploy AI and ML applications with AKS

AKS is an ideal platform for deploying and managing containerized applications that require high availability, scalability, and portability through the use of open-source tools and integration with existing DevOps processes.

Using AKS to host your AI and ML applications enables you to leverage high-performance infrastructure, cost-efficiencies, and robust security measures while maintaining the pace of innovation. AKS reduces the operational burden of running ML workloads that are optimized for the computational and infrastructure load induced by developing and deploying AI applications.
```

> ## Design and deploy AI and ML workloads on Azure
> 1. Deploy and take Flyte with an end-to-end ML orchestration solution on AKS 

Reviewer: ```suggestion
## Design and deploy AI and ML workloads on Azure

* [Deploy an application that uses OpenAI on Azure Kubernetes Service (AKS)](./open-ai-quickstart.md)
* [Deploy an AI model on Azure Kubernetes Service (AKS) with the AI toolchain operator (preview)](./ai-toolchain-operator.md)
* [Build and deploy data and machine learning pipelines with Flyte on Azure Kubernetes Service (AKS)](./use-flyte.md)

## Contributors

*Microsoft maintains this article. The following contributors originally wrote it:*

* Colin Mixon | Product Manager
* Erin Schaffer | Content Developer 2
* Brian Redmond | Principal PDM Manager
* Ken Kilty | Principal TPM
* Russell de Pina | Principal TPM

```
